keras_core/utils/backend_utils.py

```diff
@@ -49,3 +49,11 @@ def __getattr__(self, name):
             from keras_core.backend import torch as torch_backend
 
             return getattr(torch_backend, name)
+        if self._backend == "numpy":
+            # TODO (ariG23498):
+            # The import `from keras_core.backend import numpy as numpy_backend`
```

Comment:
Why is this the case?

Reply:
I am usure about this. It might have to do with `numpy` backend having a `numpy.py` file on its own (the only difference with other backends).